modules/build/src/main/scala/scala/build/interactive/Interactive.scala

```diff
@@ -0,0 +1,48 @@
+package scala.build.interactive
+
+import scala.build.Logger
+import scala.build.errors.BuildException
+import scala.build.options.BuildOptions
+import scala.io.StdIn.readLine
+
+case class Interactive(options: BuildOptions, logger: Logger) {
+
+  private val isInteractive = options.internal.interactive.getOrElse(false)
+
+  def chooseOneOptionsFromList[E <: BuildException](
```

Comment:
Typo
```suggestion
  def chooseOneOptionFromList[E <: BuildException](
```
but this could even be
```suggestion
  def chooseOne[E <: BuildException](
```